src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -121,7 +121,8 @@ services:
 ```
 
 ### Algol docker-compose
```

Comment:
You can use the tabs to easily switch between mainnet/testnet details instead of displaying both one after another, see Ethereum docs for examples

Reply:
CLI args at the top of the file are also not very readable